saleor/plugins/user_email/constants.py

```diff
@@ -80,12 +80,14 @@
 ACCOUNT_CHANGE_EMAIL_REQUEST_DEFAULT_SUBJECT = "Email change e-mail"
 ACCOUNT_PASSWORD_RESET_DEFAULT_SUBJECT = "Password reset e-mail"
 INVOICE_READY_DEFAULT_SUBJECT = "Invoice"
-ORDER_CONFIRMATION_DEFAULT_SUBJECT = "Order #{{ order.id }} details"
-ORDER_CONFIRMED_DEFAULT_SUBJECT = "Order #{{ order.id }} confirmed"
+ORDER_CONFIRMATION_DEFAULT_SUBJECT = "Order #{{ order.number }} details"
+ORDER_CONFIRMED_DEFAULT_SUBJECT = "Order #{{ order.number }} confirmed"
 ORDER_FULFILLMENT_CONFIRMATION_DEFAULT_SUBJECT = (
-    "Your order {{ id }} has been fulfilled"
+    "Your order {{ number }} has been fulfilled"
```

Comment:
I am not sure about this change @korycins

Reply:
I think this is a bug. Previously it should be `order.id`

Reply:
`order.number` should be here now